pkg/server/wire.go

```diff
@@ -213,6 +213,7 @@ var wireBasicSet = wire.NewSet(
 	wire.Bind(new(dashboardimport.Service), new(*dashboardimportservice.ImportDashboardService)),
 	plugindashboards.ProvideService,
 	comments.ProvideService,
+	wire.Bind(new(pluginsettings.Service), new(*pluginsettings.ServiceImpl)),
```

Comment:
Can you move `pluginsettings.ProvideService` together with this one so it's easy to find/reason about them?

Reply:
yes, good point